demos-europe/demosplan-core
# bug(refs DPLAN-12555): Add styling for administration email

### Ticket
DPLAN-12555

We get the email text from BE and need a parent class to style the lists correctly. 

### How to review/test
Verfahren -> Email an alle Einreichenden -> Add an email text with ordered and unordered lists -> Go to confirmation -> Lists should be styled properly.
